src/vscode-bicep/package.json

```diff
@@ -70,6 +70,12 @@
         }
       }
     },
+    "jsonValidation": [
```

Comment:
>jsonValidation [](http://example.com/codeflow?start=5&length=14)

Can we add an e2e test for the VS code extension that asserts that some completions are offered for the config file?

Reply:
Given that this involves UX scripting (we have the base setup for it, but still...), I'm ok if this particular part doesn't make it in v0.4, but it means we'll have to manually check it before the release.

Reply:
Don't have time today, will get this one in later.

Reply:
Do you mind creating an issue for that so we don't lose track of it?

Reply:
https://github.com/Azure/bicep/issues/2889